app/javascript/components/layer.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useRef, useState } from 'react'
+import Paper from 'paper';
+import consumer from '../channels/consumer';
+
+export function Layer(props) {
+  const fontSize = 20;
+
+  const pathRef = useRef(null);
+  const [penState, setPenState] = useState('pen');
```

Comment:
may want to import `CanvasTools` and utilize `CanvasTools.pen` for the initial state here

Reply:
oh ya good catch - also on a side note im not sure what state is needed after re-architecting the notebook stuff so may be best to comment this out and let you do your magic when you implement the page change stuff?